templates/server/index.html

```diff
@@ -28,34 +28,84 @@
+            href="https://wiki.ubuntu.com/{{lts_release_name}}/ReleaseNotes">
+                Read more in the release notes
+            </a></p>
         </div>
-        <div class="six-col last-col">
-            <h3>Ubuntu OpenStack</h3>
-            <ul class="list-ticks--canonical">
-                <li>Supported for five years by Canonical</li>
-                <li>Updated to the Mitaka release, including automated installation, queuing/notification and the integration of database-as-a-service</li>
-                <li>Nova LXD driver - deploy OpenStack instances as system containers, dramatically increasing per-node density</li>
-                <li>Juju OpenStack bundle -  automated deployment of OpenStack into LXD system containers</li>
-                <li>ZFS support for LXD OpenStack hosts</li>
-                <li>Certified by Microsoft to host Windows Server 2012 and Windows Server 2008 R2 as guests, under its Server Virtualisation Validation Program (SVVP)</li>
-                <li>OpenStack software, Juju, MAAS, LXD on IBM LinuxONE mainframe</li>
-            </ul>
+    </div>
+</section>
+
+<section class="row strip-light">
+    <div class="strip-inner-wrapper equal-height">
+        <div class="four-col equal-height__item equal-height__align-vertically align-center">
+            <img class="priority-0" src="{{ ASSET_SERVER_URL }}e047e34e-picto-landscape.svg"
+                alt="Landscape pictogram" width="200" height="200" />
         </div>
-        <div class="eleven-col">
-            <p><a class="external" href="https://wiki.ubuntu.com/{{lts_release_name}}/ReleaseNotes">Read more in the release notes</a></p>
+        <div class="eight-col last-col equal-height__item">
+            <h2>Landscape: Server management</h2>
+            <p>
+                Landscape allows you to manage thousands of Ubuntu machines as
+                easily as one, making the administration of Ubuntu desktops,
+                servers and cloud instances more cost-effective. It&rsquo;s
+                easy to set up, easy to use and it requires no special
+                hardware. It features:
+            </p>
+            <div class="combined-list">
+                <ul class="list-ticks four-col">
+                    <li>Management at scale</li>
+                    <li>Deploy or rollback security updates</li>
+                    <li>Kernel livepatching </li>
+                </ul>
+                <ul class="list-ticks four-col last-col">
+                    <li>Compliance reporting</li>
+                    <li>Role-based access</li>
+                    <li>Informative monitoring</li>
+                </ul>
+            </div>
+            <p>
+                <a href="https://landscape.canonical.com/"
+                    class="button--primary">
+                    <span class="external">
+                        Learn more about Landscape
+                    </span>
+                </a>
+            </p>
```

Comment:
The problem is that they have a different layout. combined list vs a single list. Not sure it would be an easy include. 
@Greggless should they all be "Kernel livepatching"